mips-lib/src/components/mips_jump_merge.rs

```diff
@@ -0,0 +1,107 @@
+// use std::fmt::Alignment;
+use log::*;
+use serde::{Deserialize, Serialize};
+use std::any::Any;
+use std::rc::Rc;
+use syncrim::common::{
+    Component, Condition, Id, Input, InputPort, OutputType, Ports, SignalValue, Simulator,
+};
+
+pub const MERGE_INSTR_ADDR_ID: &str = "merge_instr_addr_in";
+pub const MERGE_JUMP_ADDR_ID: &str = "merge_jump_addr_in";
+
+pub const MERGE_OUT_ID: &str = "merge_out";
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct JumpMerge {
+    pub(crate) id: Id,
+    pub(crate) pos: (f32, f32),
+    pub(crate) instr_addr_in: Input,
+    pub(crate) jump_addr_in: Input,
+}
+
+#[typetag::serde]
+impl Component for JumpMerge {
+    fn to_(&self) {
+        trace!("jump_merge");
+    }
+    // #[cfg(feature = "gui-egui")]
```

Comment:
dead code can be removed

Reply:
there is a bunch of random commented code scattered about, we can probably get rid of it.